language/tools/move-package/src/source_package/parsed_manifest.rs

```diff
@@ -57,14 +57,14 @@ pub struct InternalDependency {
     pub digest: Option<PackageDigest>,
 }
 
-#[derive(Debug, Clone, Eq, PartialEq)]
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
```

Comment:
Are these derives necessary anymore?

Reply:
Sorry, missed this during cleanup - clearly no longer necessary as we don't use `DependencyKind` as a key anymore. Thank you for catching this!